cmd/common.go

```diff
@@ -67,6 +67,8 @@ func runChecks(t check.NodeType) {
 	// checks that the executables we need for the node type are running.
 	binmap := getBinaries(typeConf)
 	confmap := getConfigFiles(typeConf)
+	podspecmap := getPodSpecFiles(typeConf)
```

Comment:
Is there are reason to split out pod spec and unit files from the configuration files? Maybe there is a good reason to do this but it seems like it makes it more confusing for the user trying to edit the config.yml.  

Also, although the latest version of the benchmark is the first to talk about using these pod spec files I think kubernetes was actually using this approach from at 1.7 (maybe earlier?). So I don't think it's a bad thing if people run the 1.7 or 1.6 tests and look for the config in these locations. Or am I mistaken?

Reply:
If we don't separate podspec from config files anyone running our latest kube-bench on k8s v1.7 or older will have the remediation applying to the wrong file because of what we will get for $apiserverconf.

```
Kubernetes v1.8
Remediation:
Edit the API server pod specification file $apiserverconf on the master node and set the below parameter.
--anonymous-auth=false
```

```
Kubernetes v1.7
Remediation:
Edit the $apiserverconf file on the master node and set 
the KUBE_ALLOW_PRIV parameter to \"--allow-privileged=false\"
```
But separating podspec allows me to say $apiserverpodspec for the v1.8 remediation.

Reply:
OK, that makes sense, thanks

Reply:
OK, that makes sense, thanks